astronomer/providers/core/sensors/external_task.py

```diff
@@ -128,22 +127,14 @@ class ExternalDeploymentTaskSensorAsync(HttpSensor):
 
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         super().__init__(*args, **kwargs)
-        self.hook = HttpHook(
-            method=self.method,
-            http_conn_id=self.http_conn_id,
-            tcp_keep_alive=self.tcp_keep_alive,
-            tcp_keep_alive_idle=self.tcp_keep_alive_idle,
-            tcp_keep_alive_count=self.tcp_keep_alive_count,
-            tcp_keep_alive_interval=self.tcp_keep_alive_interval,
-        )
```

Comment:
I'm not considering this a breaking change since I strongly feel this sensor is internal to Astro.